### DISCUSSION TITLE
DeepSeekCoder

### DISCUSSION THREAD

**Discussion by csmileyk** [axolotl-ai-cloud/axolotl#1171] [Q&A] [answered]
Hello,

Does Axoltl support DeepSeekCoder? Any example configs?
https://huggingface.co/deepseek-ai/deepseek-coder-6.7b-instruct

Thanks,
SmileyK

**Comment by NanoCode012** [comment, marked as answer]
Hey! Since it's a Llama based model, you may use the llama examples.

**Reply by NanoCode012** [reply]
This is still open to PR if anyone wants to curate or select better hyp for deepseek
